src/jdbc-auth.md

```diff
@@ -1383,35 +1358,30 @@ corresponding entry in the `guacamole_user_group` and
 [`guacamole_entity`](jdbc-auth-schema-entities) tables. Each user group has a
 corresponding unique name specified via `guacamole_entity`.
 
-If deleting a user group, the [corresponding entity](jdbc-auth-schema-entities)
-should also be deleted. As any user group which points to the entity will be
-deleted automatically when the entity is deleted through cascading deletion,
-*it is advisable to use the entity as the basis for any delete operation*.
+When deleting a user group, it is important to also delete the 
+[corresponding entity](jdbc-auth-schema-entities). Due to cascading deletion,
+the related user group records will be automatically deleted when the entity
+is removed.
```

Comment:
This comment now implies it, but I still think it's worth keeping the explicit recommendation to delete the associated entity as opposed to this record.